src/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -29,13 +29,13 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
             SourceGeneratorExecutionPreferenceUtilities.GetEditorConfigString));
 
     public static readonly Option2<bool> SourceGeneratorExecutionBalancedFeatureFlag = new(
-        "dotnet_source_generator_execution_balanced_feature_flag", true);
+        "dotnet_source_generator_execution_balanced_feature_flag", false);
```

Comment:
what?  why?

Reply:
This value is passed into the default value into IVsFeatureFlag
https://github.com/dotnet/roslyn/blob/4fdba2e7d431ee3888bec7f5ebbccb42c532f29f/src/VisualStudio/Core/Def/Options/FeatureFlagPersister.cs#L38
So in summary:
1. If experiment is on, feature is on
2. If experiment is off, feature is off
3. If experiment is not set or expired, feature is on
This looks like a pretty strange state.